snappy_pipeline/workflows/ngs_mapping/model.py

```diff
@@ -69,6 +69,7 @@ class TargetCoverageReportEntry(SnappyModel):
 
 
 class TargetCoverageReport(SnappyModel):
+    enabled: bool = True
```

Comment:
```suggestion
class TargetCoverageReport(ToggleModel):
    enabled: bool = True
```

Reply:
And add `ToggleModel` to the imports.

Reply:
Done (but the change is a breaking one...)